packages/example/src/pages/LinksPage.tsx

```diff
@@ -17,6 +17,7 @@ const LinksPage : React.FC = () => {
     <div><Link to={`/login`}>Login</Link></div>
     <div><Link to={`/tabs`}>Tabs Example</Link></div>
     <div><Link to='/globalUI'>Global UI</Link></div>
+    <div><Link to={`/customalert`}>Custom Alert Page</Link></div>
```

Comment:
Might be better labelling this something like "component theme override example" just to let people know the main purpose here is to demonstrate that theme overriding.